Index: app/src/main/java/com/tcc/pedal360/ActivityAlertas.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/tcc/pedal360/ActivityAlertas.java	(date 1621987587949)
+++ app/src/main/java/com/tcc/pedal360/ActivityAlertas.java	(date 1621987587949)
@@ -159,7 +159,6 @@
             bestProvider = String.valueOf(locationManager.getBestProvider(criteria, true)).toString();
             String sNumber = number.getText().toString().trim();
 
-            //You can still do this if you like, you might get lucky:
             if (ActivityCompat.checkSelfPermission(this, Manifest.permission.ACCESS_FINE_LOCATION) != PackageManager.PERMISSION_GRANTED && ActivityCompat.checkSelfPermission(this, Manifest.permission.ACCESS_COARSE_LOCATION) != PackageManager.PERMISSION_GRANTED) {
                 return;
             }
@@ -169,12 +168,11 @@
                 latitude = location.getLatitude();
                 longitude = location.getLongitude();
                 SmsManager smsManager = SmsManager.getDefault();
-                smsManager.sendTextMessage(sNumber, null, "google.com/maps/@" + String.valueOf(latitude) + "," + String.valueOf(latitude) , null, null);
-                //Toast.makeText(ActivityAlertas.this, "latitude:" + latitude + " longitude:" + longitude, Toast.LENGTH_SHORT).show();
-                //searchNearestPlace(voice2text);
+                smsManager.sendTextMessage(sNumber, null, "https://www.google.com.br/maps/@" + String.valueOf(latitude) + "," + String.valueOf(latitude) + ",17.96z", null, null);
+
             }
             else{
-                //This is what you need:
+
                 locationManager.requestLocationUpdates(bestProvider, 1000, 0, (LocationListener) this);
             }
         }
